jira_scraper/processors/text_processor.py

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
This module isn't necessary we have simple way to split text, this only adds another layer of abstraction.

Reply:
All content won't be split using `RecursiveCharacterTextSplitter` in the future. The intent is to handle different kind of content like code snippets differently. That's the reason this has been added as a separate class for extensibility.